server/server.js

```diff
@@ -194,9 +236,10 @@ var jsonResponse = function(data, res) {
 /**
  * User log POST handler.
  */
-app.post('/genotet/log', function(req, res) {
+app.get('/genotet/log', function(req, res) {
```

Comment:
Since now get/post both works with credentials, maybe we can just use post here.